mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageMapperTest.java

```diff
@@ -772,23 +772,6 @@ public void userFlagsUpdateShouldReturnCorrectUpdatedFlags() throws Exception {
     }
 
     @Test
-    public void userFlagsUpdateShouldReturnCorrectUpdatedFlagsWhenNoop() throws Exception {
```

Comment:
Why do you remove this test. Is it because as I said in the comment before you don't mind incrementing modseq for a noop operation ?
If yes sorry but I don't understand in what that does not violate the RFC